doc/MANUAL.md

```diff
@@ -440,6 +448,46 @@ extract = true
 
 Although the examples in this section show results for a single library to make them shorter and easier to read, the tool was designed to be run on entire directories and will scan all files in a directory if one is supplied.
 
+### -a INTERMEDIATE_PATH, --append INTERMEDIATE_PATH
+
+This option allows you to save the output in form of an intermediate report which you can later merge with other reports of the same format.  
+Intermediate report format
+
+```json
```

Comment:
This section reminded me: do we have the json schema for this written somewhere explicitly so we can use it for json validation?  If you don't have it yet, please file a bug to do that later.  You can take a look at the code in `test/test_json.py` for how to do the validation. We'll want that run by default if it doesn't provide a huge performance hit (and as an option if it does).